classes/Forms.js

```diff
@@ -150,4 +151,9 @@ module.exports = class Forms extends OneBlinkAPI {
 
     return super.getRequest(`/forms/${formId}`)
   }
+
+  async validateForm (form /* : Form */) /* : Promise<Form> */ {
```

Comment:
```suggestion
  async validateForm (form /* : mixed */) /* : Promise<Form> */ {
```

This should be `mixed` as what they pass in may not be a `Form`